### ISSUE TITLE
Element crashes randomly with "Trace/breakpoint trap (core dumped)"

### ISSUE THREAD

**Issue by monstro1** [vector-im/element-web#21379]
### Steps to reproduce

Seems to occur at random.

### Outcome

Crashes seemingly at random. Sometimes upon startup, other times only after it's been open for a few hours.

### Operating system

Manjaro

### Application version

Element version: 1.10.4 Olm version: 3.2.8

### How did you install the app?

Using pacman

### Will you send logs?

No

**Comment by t3chguy** [comment]
This is a bug with the community-maintained arch package

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/20926
